mp/src/game/server/momentum/mom_ruler.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "beam_shared.h"
+#include "mom_shareddefs.h"
```

Comment:
You don't need to include this, the BUFSIZ macros need to be removed in the future, you can just use 64 or something fitting.